mscore/preferences.cpp

```diff
@@ -144,6 +144,7 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_APP_PLAYBACK_PLAYREPEATS,                        new BoolPreference(true, false)},
             {PREF_APP_PLAYBACK_SPEEDINCREMENT,                     new IntPreference(5)},
             {PREF_APP_PLAYBACK_LOOPTOSELECTIONONPLAY,              new BoolPreference(true)},
+            {PREF_APP_PLAYBACK_DEFAULT_MASTER_VOLUME,              new DoublePreference(0.10)},
```

Comment:
P.S. It's the 0.10 here that gets translated to the -40dB by default

Reply:
Might be worth a corresponding coment in the code ;-)

Reply:
Oh, I'm too lazy for that at the moment :-/
How the actual dB is formulated in the code though is kind of a "round-about" way. It's not super obvious because there's like 2 different conversions that occur

Reply:
I've added it...